omero/sysadmins/windows/server-installation.txt

```diff
@@ -151,12 +152,12 @@ The following are optional depending on what services you require:
 
 .. [2] Allows larger volumes to be viewed in the :omero_plone:`Volume Viewer <volume-viewer-in-omero.web>`.
 
-PostgreSQL (8.4 or higher)
+PostgreSQL (9.2 or higher)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 PostgreSQL has to be installed and configured with PL/pgSQL and to
-accept TCP connections. PostgreSQL 8.3 and earlier are not supported.
-9.2 or later is recommended.
+accept TCP connections. PostgreSQL 9.1 and earlier are not supported.
+9.3 or later is recommended.
 
 The *One click installer* can be found on the `PostgreSQL Windows
 download page <http://www.postgresql.org/download/windows>`_. You
```

Comment:
The comment about the password here is out of place, what's it got to do with the installer?